Allow to run aiidalab image on local computer. (#54)
aiidalab/aiidalab-docker-stack: README.md

Docker images are available from Dockerhub via `docker pull aiidalab/aiidalab-docker-stack:latest`.

# Deploy on AiiDA lab server

To deploy changes, log into the AiiDA lab server and execute the following commands:
```
docker pull aiidalab/aiidalab-docker-stack:latest
docker tag aiidalab/aiidalab-docker-stack:latest aiidalab-docker-stack:latest
```
The users will gradually pick up the new image, whenever they restart their container via the _Control Panel_.

# Deploy locally

To run AiiDA lab image image locally, you first need to make sure that [Docker](https://www.docker.com/) is installed on your computer. Run `$ docker` in the command line and see if you get something like
```
Usage:	docker [OPTIONS] COMMAND

A self-sufficient runtime for containers

Options:
...
```

> If this is the case, everything proceed with the next steps. Otherwise, please visit the [Docker installation page](http://www.docker.com/install) and install Docker on your machine.

Reviewer: I'd simplify this and simply say to install docker (with link)

Author: done

> To pull image from DockerHub and start the container do the following:

Reviewer: ```suggestion
Then, pull the `aiidalab-docker-stack` image from DockerHub and start a container:
```

```
docker pull aiidalab/aiidalab-docker-stack:latest
mkdir ${HOME}/aiidalab # Create a new folder where your data vill be stored
docker run -d -p 8888:8888 -v ${HOME}/aiidalab:/home/aiida aiidalab/aiidalab-docker-stack:latest

> # Note the Docker ID that was displayed after the last command and put it into the following run

Reviewer: Wouldn't it be easier to encapsulate the steps from `docker run` in a script?

Author: done.
